src/Apps/Auction/routes.tsx

```diff
@@ -22,6 +23,50 @@ export const routes: RouteConfig[] = [
     path: "/auction-faq",
     Component: AuctionFAQ,
   },
+  {
+    path: "auction/:saleID/bid2/:artworkID",
```

Comment:
I think we should match this against bid and bid2, as done below. Removes one more change to make later on. Force will ensure that the right thing is rendered. Not much in harm in having reaction ready to handle both routes now.